external_server/models/devices.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import annotations
+import dataclasses
+
+from InternalProtocol_pb2 import Device as _Device  # type: ignore
+from InternalProtocol_pb2 import DeviceStatus as _DeviceStatus  # type: ignore
+
+
+def device_status(device: _Device, status_data: bytes = b"") -> _DeviceStatus:
+    return _DeviceStatus(device=device, statusData=status_data)
+
+
+@dataclasses.dataclass(frozen=True)
+class DevicePy:
+    """This class represents a device connected to the external server.
+
+    It contains the device identifiers and methods for assessing two devices are identical.
+    """
+
+    module_id: int
+    type: int
```

Comment:
```suggestion
    device_type: int
```

Reply:
Does this lead to any confusion?

Reply:
type is builtin class, not good practice to name something after it.